packages/playwright/src/worker/workerMain.ts

```diff
@@ -75,16 +75,20 @@ export class WorkerMain extends ProcessRunner {
 
     process.on('unhandledRejection', reason => this.unhandledError(reason));
     process.on('uncaughtException', error => this.unhandledError(error));
-    process.stdout.write = (chunk: string | Buffer) => {
+    process.stdout.write = (chunk: string | Buffer, cb?: any) => {
```

Comment:
Node types `cb` as follows:
```ts
write(buffer: Uint8Array | string, cb?: (err?: Error) => void): boolean;
write(str: Uint8Array | string, encoding?: BufferEncoding, cb?: (err?: Error) => void): boolean;
```

We should probably match the type, at least just to avoid `any`.

Reply:
I tried that, but it turned out mouthful for no real benefit.